bin/improver-blend-adjacent-points

```diff
@@ -73,6 +74,13 @@ def main():
                         help='Width of the triangular weighting function used '
                              'in the blending, in the units of the '
                              'units argument passed in.')
+    parser.add_argument('--blend_time_using_forecast_period',
+                        default=False, action='store_true', help='Flag that '
+                        'we are blending over time but using the forecast '
+                        'period coordinate as a proxy.  Note this should only '
+                        'be used when time and forecast_period and share a '
+                        'dimension: ie when all files provided are from the '
+                        'same forecast cycle.')
```

Comment:
It will throw an error.  The point of this flag is to blend using "forecast_period" for ease of use, but to treat the setup as if you were blending over time.  If you want to blend over "time" and can figure out a way to specify the point value correctly, that should work without the flag.

Reply:
I should say we could fix the plugin to deal with time - but it would be a more substantial piece of work, which is why I didn't do it for now.